Sources/PointFree/Emails/SendEmail.swift

```diff
@@ -19,7 +20,24 @@ func sendEmail(
         second(render)
     )
 
-    return AppEnvironment.current.sendEmail(
+    let headers: [(String, String)] = unsubscribeData
+      .map { userId, newsletter in
+        guard let unsubEmail = unsubscribeEmail(fromUserId: userId, andNewsletter: newsletter)
+          else { return [] }
+
+        return [
+          (
+            "List-Unsubscribe",
+            """
+            <mailto:\(unsubEmail)>, \
+            <\(url(to: .expressUnsubscribe(userId: userId, newsletter: newsletter)))>
```

Comment:
this builds up the header. the email has to be enclosed in `<mailto:...>` and the link in just `<...>`.